x-pack/plugins/monitoring/server/routes/api/v1/apm/metric_set_overview.js

```diff
@@ -15,7 +15,13 @@ export const metricSet = [
     name: 'apm_os_load',
   },
   {
-    keys: ['apm_mem_alloc', 'apm_mem_rss', 'apm_mem_gc_next'],
+    keys: [
+      'apm_mem_alloc',
+      'apm_mem_rss',
+      'apm_mem_gc_next',
+      'apm_cgroup_memory_usage',
+      'apm_cgroup_memory_limit',
+    ],
```

Comment:
We should not show the keys _Allocated Memory_ and _Proceess Total_ for the _container_ mode.
<img width="899" alt="Screenshot 2021-04-15 at 11 54 18" src="https://user-images.githubusercontent.com/5555349/114850688-55588880-9de1-11eb-8f9c-806abac6e046.png">